pkgs/top-level/all-packages.nix

```diff
@@ -6476,7 +6476,11 @@ with pkgs;
 
   buildRustCrate = callPackage ../build-support/rust/build-rust-crate.nix { };
 
-  carnix = (callPackage ../build-support/rust/carnix.nix { }).carnix_0_5_2;
+  carnix =
+    let carnix = callPackage ../build-support/rust/carnix.nix { };
+        carnixFeatures = carnix.carnix_0_6_0_features {};
+    in
+    carnix.carnix_0_6_0 carnixFeatures;
```

Comment:
It would be a nice feature to have an alias for the latest version of the crate, that is build in the Cargo.toml. Then there would be no need to always update `all-packages.nix`.